staging/src/k8s.io/cloud-provider/README.md

```diff
@@ -0,0 +1,32 @@
+# cloud-provider
+
+This repository defines the cloud-provider interface and mechanism to initialize
+a cloud-provider implementation into Kubernetes. Currently multiple processes
+use this code although the intent is that it will eventually only be cloud
+controller manager.
+
+**Note:** go-get or vendor this package as `k8s.io/cloud-provider`.
+
+## Purpose
+
+This library is a shared dependency for processes which need to be able to
+integrate with cloud-provider specific functionality.
+
+## Compatibility
+
+Cloud Providers are expected to keep the HEAD of their implementations in sync
```

Comment:
This doesn't sound like a reasonable request?

My (untested) goals for the apply repo look like this: https://github.com/kubernetes-sigs/structured-merge-diff/blob/master/RELEASE.md